Client/Frontend/UIConstants.swift

```diff
@@ -5,6 +5,12 @@
 import Foundation
 import Shared
 
+public extension UIColor {
+    class func toolbarTintColor() -> UIColor {
+        return UIColor(rgb: 0x4A4A4A)
```

Comment:
using a hex colour here makes it hard to tell if this is defining a colour that is the same as other values already defined. I think we might need to raise a separate issue about colour constants and come up with some coding conventions for defining them, as colours are starting to get confusing.